src/writeData/subscriptions/utils/form.ts

```diff
@@ -25,6 +25,9 @@ export const sanitizeForm = (form: Subscription): Subscription => {
     if (newVal) {
       form.jsonMeasurementKey.path = newVal
     }
+    if (form.jsonMeasurementKey.type === 'number') {
+      form.jsonMeasurementKey.type = 'double'
+    }
   }
```

Comment:
will this work for when we load an existing subscription into the details page? Will the dropdown still show "Number"?

Reply:
yes i've made the logic so it returns number if its double - thats what the sanitizeType function is doing.

Reply:
got it!